verilog/parser/verilog.y

```diff
@@ -230,6 +230,8 @@ is not locally defined, so the grammar here uses only generic identifiers.
 // gen_tokenizer start STRING_LITERAL
 %token TK_StringLiteral         // STRING
 %token TK_EvalStringLiteral     // STRING
+%token TK_AngleBracketInclude
```

Comment:
add comment `// STRING`, just to make it more obvious this is intended as a string

Reply:
Done.